server/src/routes/rootRouter.js

```diff
@@ -3,13 +3,14 @@ import userSessionsRouter from "./api/v1/userSessionsRouter.js"
 import usersRouter from "./api/v1/usersRouter.js"
 import clientRouter from "./clientRouter.js"
 import podcastsRouter from "./api/v1/podcastsRouter.js"
-import genresRouter from './api/v1/genresRouter.js'
+import genresRouter from "./api/v1/genresRouter.js"
 const rootRouter = new express.Router()
 
 rootRouter.use("/", clientRouter)
+rootRouter.use("/profile", clientRouter)
```

Comment:
We won't need this line since the `/profile` path has been added to the `clientRouter`